### DISCUSSION TITLE
Automatic Update does not work

### DISCUSSION THREAD

**Discussion by PH5HP** [rejetto/hfs#356] [General]
Hi,

I tried 2 times to use the autoupdate in the adminpage, but is does not work. In the console I saw something like 'the server is busy'/ I have no screenshot. Maybe others can confirm this?

The auto update is problematic for some times now, but in earlier version I could close HFS and restart it with the new HFS which was present in the HFS directory. This time that could not work, because the new version was not present in the HFS directory. I download it manually from GitHub.

Kind regards,
Henk - ftp.ph5hp.nl

**Comment by rejetto** [comment]
are you running hfs.exe under windows?
i just updated from 0.48.1 to 0.48.2 by clicking on the "check updates" button and had no problem.
I cannot imagine what could it be.
If you find more information about it let me know.

**Reply by PH5HP** [reply]
Hi,

Yes, I am running it under Windows 10. I will keep an eye on it when I get new UPDATES.. I keep you posted.

Kind regards,
Henk - ftp.ph5hp.nl
